cmd/sysl/cmd_mod_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func removeFile(t *testing.T, fs afero.Fs, file string) {
+	exists, err := afero.Exists(fs, file)
+	assert.NoError(t, err)
+	if exists {
+		err = fs.Remove(file)
+		assert.NoError(t, err)
+	}
+}
+
+func TestSyslModInit(t *testing.T) {
+	fs := afero.NewOsFs()
```

Comment:
Not too sure, but there might be a way to use a memfs or a tempfs?
That way you wouldn't need to remove files manually and it would be safe for race conditions

Reply:
I did try to look into this, but not sure how afero would work with system calls.

Reply:
ah ok